tskit_stat_benchmarks/benchmarks_no_conversion2.py

```diff
@@ -0,0 +1,66 @@
+import msprime
+import libsequence
+import allel
+import timeit
+import numpy as np
+
+NE = 1e4
+#RECRATE = 1e-8
+RHO=1e4
+RECRATE=1e3/(4*NE)
+# NU = [0.1*RECRATE, RECRATE, 10*RECRATE]
+NU = [RECRATE]
+
+def make_tree_sequence(nsam):
+    """
+    Make a tree seq w/no mutations
+    """
+    return msprime.simulate(nsam, Ne=NE, recombination_rate=RECRATE)
+
+
+def mutate_tree_sequence(ts, Nu):
+    """ 
+    Copy a tree seq and mutate it, return
+    mutated copy
+    """
+    tscopy = ts.dump_tables().tree_sequence()
```

Comment:
Just FYI, there's no need to make a copy here. Tree sequences are immutable, so you can treat them functionally for stuff like this.